frontend/src/screens/alby/SupportAlby.tsx

```diff
@@ -0,0 +1,294 @@
+        name: appName,
+        scopes: ["pay_invoice"],
+        budgetRenewal: "monthly",
+        maxAmount,
+        isolated,
+        metadata: {
+          app_store_app_id: "zapplanner",
+          recipient_lightning_address: recipientLightningAddress,
+        },
+      };
+
+      const createAppResponse = await createApp(createAppRequest);
+
+      // TODO: proxy through hub backend and remove CSRF exceptions for zapplanner.albylabs.com
+      const createSubscriptionResponse = await fetch(
+        "https://zapplanner.albylabs.com/api/subscriptions",
+        {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            recipientLightningAddress: recipientLightningAddress,
+            amount: parsedAmount,
+            message: "ZapPlanner payment from Alby Hub",
+            payerData: JSON.stringify({
+              ...(senderName ? { name: senderName } : {}),
+            }),
+            nostrWalletConnectUrl: createAppResponse.pairingUri,
+            sleepDuration: "31 days",
+          }),
+        }
+      );
+      if (!createSubscriptionResponse.ok) {
+        throw new Error(
+          "Failed to create subscription: " + createSubscriptionResponse.status
+        );
+      }
+
+      const { subscriptionId } = await createSubscriptionResponse.json();
+      if (!subscriptionId) {
+        throw new Error("no subscription ID in create subscription response");
+      }
+
+      // add the ZapPlanner subscription ID to the app metadata
+      const updateAppRequest: UpdateAppRequest = {
+        name: createAppRequest.name,
+        scopes: createAppRequest.scopes,
+        budgetRenewal: createAppRequest.budgetRenewal!,
+        expiresAt: createAppRequest.expiresAt,
+        maxAmount,
+        isolated,
+        metadata: {
+          ...createAppRequest.metadata,
+          zapplanner_subscription_id: subscriptionId,
+        },
+      };
+
+      await request(`/api/apps/${createAppResponse.pairingPublicKey}`, {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(updateAppRequest),
+      });
+
+      toast({
+        title: "Created subscription",
+        description: "The first payment is scheduled immediately.",
+      });
+
+      reloadApps();
+      setOpen(false);
+    } catch (error) {
+      handleRequestError(toast, "Failed to create app", error);
+    } finally {
+      setSubmitting(false);
+    }
+  };
+
+
+  return (
+    <>
+      <div className="h-full w-full max-w-screen-sm mx-auto flex flex-col justify-center">
+        <div className="flex flex-col items-center justify-center gap-6">
+          {!subscription &&
+            <>
+              <section className="text-center">
+                <h2 className="text-3xl font-semibold mb-2">
+                  ✨ Your Support Matters
```

Comment:
In mobile view, there is no spacing, everything feels too tight in my opinion

![image](https://github.com/user-attachments/assets/473c7d7e-10fc-43fc-95ea-dcfdc81fccdb)

Reply:
fixed